plugins/trace_api_plugin/trace_api.swagger.yaml

```diff
@@ -1,12 +1,13 @@
 openapi: 3.0.0
 info:
   title: Trace API
+  description: This plubing must specifiy one option `trace-rpc-abi` or `no-trace-abis`. These options are provided at nodeos startup on the command line or in config.ini file. Configuration option `trace-rpc-abi` There must be at least one ABI specified OR the flag trace-no-abis must be used. ABIs are specified as "Key=Value" pairs in the form <account-name>=<abi-def> Where <abi-def> can be an absolute path to a file containing a valid JSON-encoded ABI a relative path from `data-dir` to a file containing a valid JSON-encoded ABI. Configuration option `trace-no-abis` use to indicate that the RPC responses
```

Comment:
This repeats the same description of the `--trace-rpc-abi` option in the [trace api docs](https://docs.eosnetwork.com/leap/latest/nodeos/plugins/trace_api_plugin/#configuration-options). We can just link to the specific configuration option if needed to avoid maintaining duplicate sources of truth.